### PR TITLE
Playground setup

### PR DESCRIPTION
**Pull request by gksander**
This PR sets up the tooling to start building out a Sanity plugin for groqd playground. Nothing super pretty here, but wanted to make sure the build/release process was working so we can hit the ground running with the actual playground.

### CODE REVIEW

File: packages/groqd-playground/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "groqd-playground",
```
Comment (gksander) [inline packages/groqd-playground/package.json:2]: Thoughts on name before we ship the first release?

Comment (carloskelly13) [reply packages/groqd-playground/package.json:2]: Either `playground` or `explorer`

**Review by carloskelly13 — APPROVED** [review]

File: packages/groqd-playground/src/Playground.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
```
Comment (carloskelly13) [inline packages/groqd-playground/src/Playground.tsx:1]: Tiny thing, but can't we configure React to be globally included?

File: packages/groqd-playground/tsup.config.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+	entry: ["src/index.ts"],
```
Comment (carloskelly13) [inline packages/groqd-playground/tsup.config.ts:4]: Tiny thing but spacing seems off here.

Comment (gksander) [reply packages/groqd-playground/tsup.config.ts:4]: oh yeah, you're right – I wonder why ESLint didn't pick this up 🤔 I'll take a peak.

Comment (gksander) [reply packages/groqd-playground/tsup.config.ts:4]: `eslint packages/**/src --quiet` there it is, only checking against `src` files, whoops.

**Pull request merged**